internal/servers/controller/controller.go

```diff
@@ -178,6 +178,24 @@ func (c *Controller) Start() error {
 }
 
 func (c *Controller) registerJobs() error {
+	if err := c.registerSessionCleanupJob(); err != nil {
```

Comment:
Might be good to merge `main` into `vancluever/session-cleanup` and then `vancluever/session-cleanup` into this branch as main now makes use of this method.

Reply:
Going to rebase this to main post-release actually, as that "long-running" branch is not being used anymore.

Reply:
This was done last week (as discussed with @louisruch offline so that we could resolve some job throttling issues).